### ISSUE TITLE
Do not request preview for non-previewable file types

### ISSUE THREAD

**Issue by kernins** [owncloud/core#24489]
```
"NetworkError: 404 Not Found - https://myclouddomain/index.php/core/preview.png?file=%2Faeroex.pdf ..."
```

Currently `Files` frontend requests preview for each and every file, which is pretty pointless imo and also adds unnecessary load.

Most nice and future-proof approach would be to export "previewable" (supported by backend) filetypes into FE context at runtime and then request preview only for matching files. Admin setting would also be great

**Comment by kernins** [comment]
ah, forgot to mention oc version - 9.0.2

**Comment by RealRancor** [comment]
@kernins This is a duplicate of https://github.com/owncloud/core/issues/24216

**Comment by kernins** [comment]
@RealRancor not exactly.

My suggestion is not about just suppressing those 404's in any way - they are more or less sematically correct in this situation. It is about preventing FE (frontend) from making those pointless requests in the first place. This is the only proper way to fix this, anything other is workaround.

Only supported files should have
```
background-image: url("/index.php/core/preview.png?file=...");
```
and all others should use statics like
```
background-image:url(/core/img/filetypes/application-pdf.svg)
```

**Comment by RealRancor** [comment]
@kernins Its the same issue. Just post your suggestion there. No need to have two issues open about this

**Comment by kernins** [comment]
Sure. Just didn't though about it that way)

Closed as duplicate of #24216

**Comment by RealRancor** [comment]
I think your solution is even better so the other issue benefits from that suggestion
